sdv/multi_table/hma.py

```diff
@@ -418,11 +419,9 @@ def _model_tables(self, augmented_data):
             augmented_data (dict):
                 Dictionary mapping each table name to an augmented ``pandas.DataFrame``.
         """
-        parent_map = self.metadata._get_parent_map()
```

Comment:
why are we getting rid of this check?

Reply:
We need to model all the tables now to get default parameters, not just the roots.